recording/lib/reporter.ex

```diff
@@ -160,53 +165,17 @@ defmodule Membrane.RTC.Engine.Endpoint.Recording.Reporter do
 
   defp add_wallclock_start_time(track, rtcp) do
     delta_t_ns =
-      (rtcp.sender_info.rtp_timestamp - track.start_timestamp) / track.clock_rate * @sec_to_ns
+      if rtcp_overflow?(rtcp.sender_info.rtp_timestamp, track.start_timestamp),
+        do:
+          (rtcp.sender_info.rtp_timestamp - track.start_timestamp + @rtp_timestamp_size) /
+            track.clock_rate * @sec_to_ns,
+        else:
+          (rtcp.sender_info.rtp_timestamp - track.start_timestamp) / track.clock_rate * @sec_to_ns
```

Comment:
I think this way it would be easier to understand.
```suggestion
      if rtcp_overflow?(rtcp.sender_info.rtp_timestamp, track.start_timestamp),
        do:
          rtcp.sender_info.rtp_timestamp - track.start_timestamp + @rtp_timestamp_size,
        else:
          rtcp.sender_info.rtp_timestamp - track.start_timestamp
          
       delta_t_ns = delta_t_ns / track.clock_rate * @sec_to_ns
```